pkgs/development/libraries/dpp/default.nix

```
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libopus
+, openssl
+, zlib
+, libsodium
+, gcc
+, pkg-config
+, autoPatchelfHook
+, lib
+}:
+let
+   libExt = if stdenv.hostPlatform.isDarwin then ".dylib" else ".so";
+in
+stdenv.mkDerivation rec {
+  pname = "dpp";
+  version = "10.0.29";
+
+  src = fetchFromGitHub {
+    owner = "brainboxdotcc";
+    repo = "DPP";
+    rev = "v${version}";
+    sha256 = "BJMg3MLSfb9x/2lPHITeI3SWwW1OZVUUMVltTWUcw9I=";
```

```suggestion
    hash = "sha256-BJMg3MLSfb9x/2lPHITeI3SWwW1OZVUUMVltTWUcw9I=";
```
please don't strip sri hashes